### PR TITLE
Settings: Adjust to sentence case

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

A few headings were using title case

Remove experimental from "Follow renames in file history (experimental)". While the functionality could be improved, this is how Git is implemented.

Seen when updating the doc for 4.1 (would like to update the doc with this also if this is not included in 4.1).

## Test methodology 

Manual

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by RussKie — APPROVED** [review]

**Comment by RussKie** [comment]
Please target the release/4.1 branch.

File: appveyor.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: 4.1.0.{build}
```
Comment (pmiossec) [inline appveyor.yml:1]: @gerhardol This change should not have been part of this PR, right!?
We have to revert it?

Comment (gerhardol) [reply appveyor.yml:1]: yes

Comment (gerhardol) [reply appveyor.yml:1]: Due to rebasing 
I still prefer to cherry-pick from master to the release branches...

**Pull request merged**